output/esp32c2.yaml

```diff
@@ -0,0 +1,66 @@
+name: esp32c2
+variants:
+  - name: esp32c2
+    cores:
+      - name: main
+        type: riscv
+        core_access_options: !Riscv {}
+    memory_map:
+      - !Nvm
+        range:
+          start: 0x0
+          end: 0x1000000
+        is_boot_memory: true
+        cores:
+          - main
+      - !Ram
+        range:
+          start: 0x403cc800
```

Comment:
```suggestion
          start: 0x40380000
```

Reply:
@MabezDev This should be the start address of the Internal SRAM 1 or start at [`ORIGIN + LENGTH`](https://github.com/esp-rs/esp-flash-loader/pull/4/files#diff-89cb8d65f6ac20bb5eb03286f00518627ba185cd29f2f41cb43815e6023456ebR3) (values defined in `ld/esp32c2.x`) and ends at the end of the IRAM section. 

The `0x403cc800` is clearly wrong because it doesn't work, so I would like to understand where it comes from and update the readme